mender-image-prepare.sh

```diff
@@ -0,0 +1,535 @@
+#!/bin/bash
+
+show_help() {
+cat << EOF
+Usage: $0 [prepare_image | make_sdimg | make_sdcard] [-i|--image | -d|--device -r|--rootsize]
+
+        device - can be /dev/sd* or /dev/mmcblk*
+
+        prepare_image - shrinks existing embedded image
+                        obligatory parameters: --image
+
+        make_sdcard   - formats, partitions and sets the sd card
+                        obligatory parameters: --image & --rootsize
+                        
+                        note: rootsize can found as an output from
+                        'prepare_image' command
+
+        make_sdimg    - composes image file compliant with Mender partition
+                        table scheme
```

Comment:
I didn't really understand the difference between `make_sdcard` and `make_sdimg`. Is one for the image file, and one for the physical card?

Reply:
Exactly, just for convenience.

Reply:
would say "partition layout", we use that in docs (not scheme)

Reply:
But why is an explicit operation for the physical card needed? Won't the user just convert the whole image, and then flash the entire thing in one go to the physical card?

Reply:
@estenberg: I will use "partition layout" then.
@kacf: "explicit operation for the physical card" - we can think of removing that path later...